### PR TITLE
Feature/507 635 Integration Tests

### PR DESCRIPTION
**Pull request by ggrg**
Integration tests for central-settlement:
```
1..67
# tests 67
# pass  67

# ok
```

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "PORT": 3007,
   "HOSTNAME": "http://central-settlements.local",
-  "DATABASE_URI" : "mysql://central_ledger:password@localhost:3306/central_ledger",
+  "DATABASE_URI" : "mysql://central_ledger:cVq8iFqaLuHy8jjKuA@localhost:3306/central_ledger_integration",
```
Comment (rmothilal) [inline config/default.json]: Please revert this change

File: package.json [hunk 2]
```diff
@@ -23,8 +23,8 @@
         "@mojaloop/central-services-database": "3.1.0-snapshot",
         "@mojaloop/central-services-error-handling": "0.5.1",
```
Comment (rmothilal) [inline package.json]: Please update version to 5.2.0 and run npm install to update package-lock.json

Comment (ggrg) [reply package.json]: done

File: config/default.json (hunk 1 above)
Comment (ggrg) [reply config/default.json]: done
